### PR TITLE
Add optional parameter read as json in Read Text File action

### PR DESCRIPTION
**Pull request by Oytizzo**
## PR Type

Add Feature `read as json` for `Read Text File` common action

## PR Checklist

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made.
- [x] Version number has been updated.
- [x] No extra module was necessary. Required modules have been added to respective "requirements*.txt" files.
- [x] (Team) Label with affected action categories and semver status.

## Overview

This version gives the capability to read a .json file as text file and then convert the text into json object.

There won't be any breaking to the existing code or test cases.

### CODE REVIEW

File: Framework/Built_In_Automation/Sequential_Actions/common_functions.py [hunk 1]
```diff
@@ -2827,17 +2827,35 @@ def save_text_from_file_into_variable(data_set):
...
         import PyPDF2
         text_file_path = ""
         var_name = ""
         var_value = ""
+        read_as_json = False
 
         for left, mid, right in data_set:
             if left.strip().lower() == "file path":
                 text_file_path = CommonUtil.path_parser(right.strip())
+            elif left.strip().lower() == "read as json" and right.strip().lower() == "true":
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Sequential_Actions/common_functions.py]:
```python
 right.strip().lower() in ("true","yes","ok","enable","accept")
 ```

File: Framework/Built_In_Automation/Sequential_Actions/common_functions.py [hunk 2]
```diff
@@ -2866,6 +2884,8 @@ def Read_text_file(data_set):
             with open(text_file_path, "r") as file:
                 data = file.read()
                 var_value += data
+                if text_file_path.endswith("json") and read_as_json:
+                    var_value = json.loads(var_value)
 
         sr.Set_Shared_Variables(var_name, var_value)
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Sequential_Actions/common_functions.py]:
```python
return sr.Set_Shared_Variables(var_name, var_value)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 # Version 14
 
+### [14.11.1] [March 13, 2022]
```
Comment (Muntasib-creator) [inline CHANGELOG.md]: please revert the lines. we will do versioning when merging into beta

File: Framework/Version.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
 [ZeuZ Python Version]
-version = 14.10.1
```
Comment (Muntasib-creator) [inline Framework/Version.txt]: please revert the lines. we will do versioning when merging into beta

**Review by Muntasib-creator — COMMENTED** [review]
please revert the version lines. we will do versioning when merging into beta
and also please make the changes as mentioned in the function

**Review by sazid — APPROVED** [review]

**Review by Muntasib-creator — APPROVED** [review]

**Pull request merged**